external_search.py

```diff
@@ -602,13 +608,13 @@ def _search():
         def _works():
             return self.query_works(
                 self.test_search_term, filter=None, pagination=None,
-                debug=False
+                debug=True
```

Comment:
Without this, we don't have .sort_title and .sort_author to display when we render the results.